src/test/kotlin/com/terraformation/backend/seedbank/search/SearchServiceBasicSearchTest.kt

```diff
@@ -0,0 +1,201 @@
+package com.terraformation.backend.seedbank.search
+
+import com.terraformation.backend.customer.model.Role
+import com.terraformation.backend.db.default_schema.FacilityId
+import com.terraformation.backend.db.seedbank.AccessionId
+import com.terraformation.backend.db.seedbank.AccessionState
+import com.terraformation.backend.db.seedbank.tables.pojos.BagsRow
+import com.terraformation.backend.search.FieldNode
+import com.terraformation.backend.search.NoConditionNode
+import com.terraformation.backend.search.SearchDirection
+import com.terraformation.backend.search.SearchFilterType
+import com.terraformation.backend.search.SearchResults
+import com.terraformation.backend.search.SearchSortField
+import io.mockk.every
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+internal class SearchServiceBasicSearchTest : SearchServiceTest() {
```

Comment:
One area I still questions a little bit is this class. These are the "leftover" tests after I split out the other classes. They seems to exercise the basic search facilities of the application, for the most part, which was the reason for the name.